src/twisted/conch/test/test_keys.py

```diff
@@ -682,6 +758,25 @@ def test_fromBlobDSA(self):
         self.assertTrue(dsaKey.isPublic())
         self.assertEqual(dsaPublicData, dsaKey.data())
 
+    def test_fromBlobECDSA(self):
+        """
+        A public ECDSA key is correctly generated from a public key blob.
```

Comment:
Here, since you're not saying "test that", it's a bit more subtle; "`Key.fromString` loads elliptic curve keys from blobs", perhaps?